frontend/src/App.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+
+import { default as Login } from './components/login';
```

Comment:
Does `import Login from './components/login'` not work? I haven't seen someone do aliasing for the default export of module before.